bittide-instances/src/Bittide/Instances/ClockControl.hs

```diff
@@ -22,4 +22,7 @@ callisto3 ::
   -- | Speed change requested from clock multiplier
   Signal Basic200 SpeedChange
 callisto3 clk rst ena dataCounts =
-  callistoClockControl clk rst ena config dataCounts
+  callistoClockControl clk rst ena config availableLinkMask dataCounts
+ where
+  -- all links available
+  availableLinkMask = pure $ pack $ repeat high
```

Comment:
(I think I overheard Martijn and/or Lucas already having a discussion already having a discussion over this last week.)

But I'm not a fan of dragging Vecs into this, since it creates terrible VHDL:

```VHDL
  \c$vec\ <= (Blaat_topEntity_types.array_of_std_logic'(0 to 9-1 =>  ('1') ));

  -- map begin
  r_map : for i in result_res_res'range generate
  begin
    result_res_res(i) <= (std_logic_vector'(0 => \c$vec\(i)));


  end generate;
  -- map end

  -- concatBitVector begin
  concatBitVectorIter_loop : for i_0 in 0 to (9 - 1) generate
    result_res(((i_0 * 1) + 1 - 1) downto (i_0 * 1)) <= std_logic_vector'(result_res_res(result_res_res'high - i_0));
  end generate;
  -- concatBitVector end

  result <= result_res;
```

Compared to something like `maxBound` or `complement 0` which just turns into a constant in VHDL.

Reply:
See my argument in https://github.com/bittide/bittide-hardware/pull/226#discussion_r1069179350.

Reply:
`complement 0` however does not assume Integer semantics. I'm fine with that one too.